ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,236 @@
+missed a meeting, you can also read the mailing list archives to see what we
+went over in the past. This is a low volume mailing list (2 emails per week)
+and you can unsubscribe at any time.
+
+[[Read more about other mailing lists available to join|doc
+staff/mailing-lists]]
+
+### In-person meetings
+
+We hold two meetings every Monday:
+
+#### Board of Directors (BoD)
+
+**Mondays 7-8pm**
+
+BoD is where we discuss and vote on administrative decisions. If you are
+interested in gaining insight into the internal operations the OCF from an administrative
+perspective, please come to BoD meetings! Meetings always are open to all.
+
+#### Staff Meeting
+
+**Mondays 8-9pm**
+
+In these meetings, all OCF staff (that includes you!) get together to discuss
+technology, learn from each other, and work on OCF projects. These meetings will
+be a mix of presentations, work time, and socializing. Our goal is to build OCF
+staff into a strong community of capable individuals.
+
+### Contribute
+
+We know that finding ways to contribute to the OCF can feel daunting, but we
+don’t want you to be discouraged. Don’t worry about finding the best way to
+contribute, as long as you find something that you want to work on. Here are
+some ways to get started:
+
+#### Projects
+
+We have a curated list of projects at [ocf.io/projects][projects]. Projects
+labeled “newbie” are ones we think are best fit for new staffers, but we are
+happy to provide context and mentorship on any project. In general, don’t worry
+about fully understanding the project description. If you find a something that
+interests you, respond to the thread or reach out to a veteran staffer to get
+more information.
+
+#### GitHub Issues
+
+All of our code is on GitHub ([github.com/ocf][ocf-github]). Sometimes you can find ways to
+contribute by reading open issues, but they can sometimes be hard to understand
+since they’re short on context. Some issues are also marked as “good first
+issue.” Here’s a list of our most active repos, with some examples of things we
+can do with them:
+
+* [puppet][puppet] ([issues][puppet/issues]) - contains Puppet modules, which
+  controls the software and configuration on all of our servers.
+   * Install software packages on the desktops
+     ([ocf_desktop/manifests/packages.pp][puppet-desktop-packages])
+   * Configure Firefox to show the "are you sure you want to quit" dialog
+     ([PR #373][puppet-373])
+   * Allow operations staff to refund pages ([PR #368][puppet-368])
+   * Monitor our mirrors to see if they’re out of date
+     ([healthcheck script][mirror-healthcheck],
+     [configuring server to call the healtcheck
+     script][mirror-healthcheck-puppet],
+     [Prometheus rules for alerting][prometheus-mirror])
+   * Monitor printers to see if they’re out of paper, toner, or jammed
+     ([printer.rules.yaml][prometheus-printer])
+   * Send a desktop notification if a print jobs fails ([PR #321][puppet-321])
```

Comment:
If I were a newcomer, reading through these three examples, I might think that Puppet's role is to monitor services or send notifications to desktops. These examples can be at the least reworded to emphasize what Puppet's role in enabling each of these is.

(For that matter, newcomers probably haven't heard of Prometheus fwiw)

Reply:
changed "Prometheus" to "Monitoring"